build.sbt

```diff
@@ -110,7 +103,37 @@ val browserTestSettings = Seq(
       },
       org.scalajs.jsenv.selenium.SeleniumJSEnv.Config().withKeepAlive(debugging)
     )
-  }
+  },
+  downloadLatestChromeDriver := {
+    if (java.nio.file.Files.notExists(new File("target", "chromedriver").toPath)) {
```

Comment:
I'm afraid this won't work on Windows, where filename is `chromedriver.exe`

Reply:
Good point, do we have anybody with windows to fix this?:)